### ISSUE TITLE
linking date object

### ISSUE THREAD

**Issue by eroux** [buda-base/owl-schema#108]
DILA has a whole database of dates (http://authority.dila.edu.tw/time/) that they have links to in their data. They are currently converting these to xsd:dates, but in a second step they want to link to the date object (once it will be converted to RDF). I'm not entirely sure how to do that...

(as an unrelated side note, it seems date indications in SEA manuscripts often do not contain a year indication, just day of week, day of month and month...)

**Comment by eroux** [comment]
in the meantime the proposal is to add `bdo:onDateDILATimeID` which is a temporary fix...

**Comment by xristy** [comment]
With the addition in bdo.ttl of the bdo:DateType and subclasses bdo:ChineseDateType, etc, and the object properties: bdo:duringReign and bdo:duringDynasty with ranges bdo:ReignOfRuler and bdo:ChineseDynasty there's a basic bit of modeling that can be extended. 

The bdo:DateType is added to the union of xsd:date, xsd:integer which covers the possibilities for bdo:notAfter, bdo:notBefore and bdo:onDate.

There needs to be a property, bdo:ruler, on bdo:ReignOfRuler with range bdo:Person which I'll add if this approach is in line with your thinking.

wdyt?

**Comment by eroux** [comment]
Oh I wasn't thinking of modeling the dates just yet, just having the possibility for DILA to indicate their ID of dates in the person data. When/if they start to work on their date database, maybe we can discuss modeling the dates then?

**Comment by xristy** [comment]
I thought it would be sufficient to add bdo:DateType to the bdo:IntegerOrDate union which is the range of bdo:onDate, bdo:notBefore and bdo:notAfter. The other items were based on looking at the DILA site. Do they need to be removed?

**Comment by eroux** [comment]
Well... maybe it would be wise to keep them in a separate branch or remove them yes, as it may introduce a bit of confusion... It will be something interesting to model in the future though, I hope we can make time for that at some point

**Comment by xristy** [comment]
closed via https://github.com/buda-base/owl-schema/commit/4386c0aac8298e994232c03c02697f52092558de

**Comment by eroux** [comment]
I'm wondering why we're considering dates like types and not entities? (or facets or whatever but not type)

In any case, this debate may be long so it's probably worth not having it. In the meantime, I'm thinking we should name the class `bdo:Date` instead of `bdo:DateType`, because its individuals are dates, not date types. This would be on the same model as:
- `:Language`, which individuals are languages, not language types
- `:LineageType` which individuals are lineage types, not lineages

what do you think?

**Comment by xristy** [comment]
I did several pertinent renames, including bdo:Date and so on.

I don't see that the discussion need be protracted.

I would push back on making bdo:Date a bdo:Entity since my understanding of the meta-model is that bdo:Entity is supposed to be the class of resources that constitute the primary _things_ in the buda domain of discourse. Maybe a bdo:Facet is ok. 

I undertstand bdo:Facet to be originally intended to model what are usually clusters of properties - like bdo:AgentInRole that may be used in faceting search results. In that line of thinking then bdo:Date as a bdo:Facet is perfectly reasonable. 

I assigned it originally to bdo:Type as thinking of it analogous to xsd:date which wasn't very inventive
